src/components/Player/Player.tsx

```diff
@@ -161,6 +172,23 @@ const Player: React.FC<Props> = ({
 
       playerRef.current = window.jwplayer(playerElementRef.current) as JWPlayer;
 
+      // Add user_id and profile_id for CDN analytics
+      const { sources } = item;
+
+      sources?.map((source) => {
```

Comment:
I think we should make these calculations part of the service / controller, either ApiService or part of the new AnalyticsController responsible for the calculations. That would help to reuse this logic for other platforms in the future. @dbudzins do you have a preference?